src/components/ListView/ListViewDataSource.js

```diff
@@ -0,0 +1,194 @@
+import invariant from 'fbjs/lib/invariant'
+
+export default class ListViewDataSource {
```

Comment:
react native is changing their fbjs requires to work like standard npm requires. you can add `/* eslint disable */` to the top of the file as is done in other modules mirroring RN